about.html

```diff
@@ -0,0 +1,129 @@
+      </nav>
+    </nav>
+  </header>
+  <main>
+    <section id="headlineAbout">
+      <div class="container-main cnt-abt">
+        <p class='title-tag ff-2 clr-2 al-center'>"Hello! Digital World"</p>
+        <h1 class="about-head ff-1 clr-2 al-center" id="About-title">
+          STOCK MARKET<br>GLOBAL SUMMIT 2023
+        </h1>
+        <div class="abt-desc" id="abt-main-desc">
+          <p class="sec-text ff-2 al-center" id="desc-text">
+            The need for Financial freedom, Stock Market and Trading. 
+            Here from global financial experts on the best practices
+            to achieve goal.
+          </p>
+          <div class="pattern-2" id="desc-pat-tr-2"></div>
+        </div>
+        <p class="al-center" id="duration">
+          For More Details and participation, please contact us at <br>
+          DW-summit;gmail.com
+        </p>
+      </div>             
+    </section>
+    <section class="abt-sec">
+      <div  class="container-main abt-logo cnt-abt">
+        <h2 class="head4 ff-2 al-center">
+         STOCK MARKET<br>GLOBAL SUMMIT 2023 LOGO
+        </h2>
+        <hr>
+        <img src="images/dw.jpg" alt="DW Logo" id="brand-logo">
+      </div>
+    </section>
+    <section class="abt-sec">
+      <div class="container-main abt-logo cnt-abt" id = 'past-summit'>
+        <h2 class="head4 ff-2 al-center">
+          See the past DW Global Summits
+        </h2>
+        <hr>
+        <div class="summit-cnt">
+          <div class="summits" id="summit-1">
+            <h2 class="clr-4">
+              2021
+            </h2>
+            <p class="clr-4">
+              DW Global Summit 2021 in India
+            </p>
+          </div>
+          <div class="summits" id="summit-2">
+            <h2 class="clr-4">
+              2022
+            </h2>
+            <p class="clr-4">
+              DW Global Summit 2022 in USA
+            </p>
+          </div>
+        </div>
+        
+      </div>
+    </section>
+  </main>
+  <footer class="footer bg-dark" id="abt-footer">
+    <div class="container-main mob-only" id="abt-partners-ph">
+      <h3 class="head3 clr-4 fw" id="partners">Partners</h3>
+      <hr>
+      <div class="partners-list">
+        <div class="partners">
+          <img src="images/ec.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/offsec2.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/microsoft-r-logo-png-12.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/cisco.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/mcAfee.png" alt="">
+        </div>
+      </div>      
+    </div>
+    <div class="container-main" id="host-ph">
+      <div id="logo"><a href="#"><img src="images/dw.jpg" alt=""></a></div>
+      <div class="rh"><small>2015 DIGITAL WORLD (India).<br> All Rights Reserved</small></div>
```

Comment:
- [ ]  You should provide the alt value so that in case the image does not appear, the user can know what was in place of an image.  This will improve your page accessibility as well.